app/assets/javascripts/bootstrap-select.js

```diff
@@ -303,6 +303,7 @@
     style: 'btn-default',
     size: 'auto',
     title: null,
+    counter: false,
```

Comment:
Added a custom option to bootstrap-select "counter" (boolean) which adds a badge counter to the title of the selectpicker